app/lib/cache_key.rb

```diff
@@ -0,0 +1,10 @@
+class CacheKey
+  def self.generate(identifier)
+    sha = CODE_REVISION_FOR_CACHE
+    feature_flags_last_changed = Feature.maximum(:updated_at)
```

Comment:
Thinking around this, does this mean we're doing a db lookup on every cache key generation/lookup, could be detrimental in terms of speed possibly 🤔

Reply:
A little ironic perhaps, but `feature_flags_last_changed` itself could be cached for a few minutes. :inception:

Reply:
i think the cost of this query is going to be negligible, and the benefits are worth having?

Reply:
We can go with this for now and see how it performs under the load test